src/components/AmsterdamLogo/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { LogoShort } from '@datapunt/asc-assets';
+import styled, { css } from 'styled-components';
+import { breakpoint, themeSpacing, themeColor } from '@datapunt/asc-ui';
+
+export const srOnlyStyle = () => ({ srOnly }) =>
+  srOnly
+    ? css`
+        border-width: 0;
+        clip: rect(0, 0, 0, 0);
+        height: 1px;
+        margin: -1px;
+        overflow: hidden;
+        padding: 0;
+        position: absolute;
+        width: 1px;
+      `
+    : '';
+
+export const LogoStyle = styled(LogoShort)`
+  display: block;
+  height: 100%;
+  width: 100%;
+`;
+
+export const LogoTitleStyle = styled.span`
+  ${srOnlyStyle()}
+`;
+
+const AmsterdamLogoStyle = styled.a`
+  display: inline-block;
+  height: 30px;
+  width: 68px;
+  flex-shrink: 0;
```

Comment:
`flex-shrink` Doesn't have any effect when the element isn't a flex element. In this case it's `inline-block`, not `flex` or `inline-flex`.

Reply:
correct, I missed that,removed.